ivy/__init__.py

```diff
@@ -463,7 +463,7 @@ def with_rank_at_most(self, rank):
         else:
             return self
 
-    def as_shape(shape):
```

Comment:
I guess it is reasonable to have this function as static.

Reply:
Ok, sir. I will retrieve these 2 changes. 👍